package/Runtime/Scripts/Internal/NimbusIOSAdManager.cs

```diff
@@ -37,6 +37,11 @@ internal void SetAdUnit(NimbusAdUnit adUnit)
 
         #region iOS Event Callbacks
 
+        // TODO missing response data, i need to the Nimbus response object back
+        // private void onAdResponse(....) {
+        //     
+        // }
+        
```

Comment:
@brugg realized that IOS is missing the Nimbus response data. It's required, review the onAdResponse implemented on Android.